data_pipeline/data_processing/aggregatedcsv2redis.py

```diff
@@ -42,8 +42,8 @@ def __init__(self, filename: str):
             logger.info('{} only have {} row.'.format(
                 filename, self.data.shape[0]))
         # Add candidate ID column. candidate ID is <Ballot Item>;<CandidateControlledName>;<Election Date>
-        self.data['ID'] = self.data['Ballot Item'].map(str) + ';' + self.data['CandidateControlledName'].map(
-            str) + ';' + self.data['Election Date'].map(str)
+        self.data['ID'] = self.data['Ballot Item'].map(
```

Comment:
It seems like the Ballot Item can also have spaces in it: "Councilmember-District 4;David-Cohen;11/3/2020" Can we replace the spaces there as well?

Reply:
Yes, will do.